mobify/platform-scaffold
# [Integration manager] sfcc confirmation

This PR adds the confirmation page to the SFCC connector. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-49
 **Linked PRs**: n/a

## Changes
- Added a fetchCheckoutConfirmationData command for SFCC
- Separated updateShippingAndBilling commands into separate ones (merlins)
- Added updateShipping and updateBilling commands (SFCC)
- Added users email address to redux state
- Fixed default value for billing address same as shipping checkbox

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to your cart and complete the checkout
- the confirmation page still works as expected
-  Preview [the SFCC site](https://preview.mobify.com/?url=http%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to your cart and proceed through the checkout to the payment page
- Enter correct information on the page and place the order
- the confirmation page on SFCC should work as expected
